css/style.css

```diff
@@ -0,0 +1,78 @@
+body {
+
+
+  background-color: black;
+  font-color: white;
+  font-family: serif;
+  font-size: 18px;
+  border-top: 5px solid #222222;
+  margin-top: 20px;
+  margin-left: 70px;
+}
+
+h1{
+
+  color: white;
+  font-style: italic;
+  font-size: 50px;
+  text-align: center;
+}
+
+span.bold{
+
+  font-weight: bold;
+  font-size: 65px;
+}
+
+h2 {
+
+  color: red;
+}
+
+p{
+
+  margin-right: 60px;
+  color:white;
+
+}
+
+p.other{
+
+  margin-right: 0px;
+}
+
+h4{
+color: red;
+font-style: sans-serif;
+text-align: center;
+
+}
+
+h5{
+
+  font-color: white;
+}
+
+img {
+  float: right;
+  border: lightgray 1px solid;
+  margin-left: 40px;
+  }
+
+  img.2 {
```

Comment:
:hammer: What are you trying to style here? Maybe I can help with the selector. As it stands, this is trying to style all images with a class of 2, but we can't use 2 as a class name. It has to start with a letter but can include numbers, such as `.image2`.